sunpy/coordinates/wcs_utils.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+import astropy.wcs.utils
+
+from .frames import *
+
+__all__ = ['solar_wcs_frame_mapping']
+
+def solar_wcs_frame_mapping(wcs):
+    """
+    This function registers the coordinates frames to their FITS-WCS coordinate
+    type values in the `astropy.wcs.utils.wcs_to_celestial_frame` registry.
+    """
+
+    xcoord = wcs.wcs.ctype[0][0:4]
+    ycoord = wcs.wcs.ctype[1][0:4]
```

Comment:
Looking at the WCSLIB source code (https://github.com/astropy/astropy/blob/master/cextern/wcslib/C/wcs.c#L2438) I think that ``WCSSUB_CELESTIAL`` is defined as an alias for ``WCSSUB_LONGITUDE`` and ``WCSSUB_LATITUDE``, and these are defined as anything that matches:

* ``RA--*``, ``?LON*``, or ``??LN*`` for the longitude
* ``DEC-*``, ``?LAT*``, or ``??LT*`` for the latitude

Does this covert the solar ctypes adequately? (it won't match e.g. heliocentric cartesian, but that's normal)